packages/mds-orm/connection-manager.ts

```diff
@@ -0,0 +1,39 @@
+import { Connection, createConnections, getConnectionManager, ConnectionOptions } from 'typeorm'
+import { ConnectionName, Connections } from './connections'
+
+let connections: Connection[] | null = null
+
+export const ConnectionManager = (options: ConnectionOptions[] = Connections()) => {
```

Comment:
nit - I'm a little surprised not to see any logging or error handling in this function. What if our apps can't connect to the db or overload the db by creating too many connections? Up to you of course, just thought I'd mention.